### ISSUE TITLE
Add WCAG properties to the Accessibility Attribute

### ISSUE THREAD

**Issue by jshslsky** [ucla-oarc-mobile/casa-on-rails#83]
- The setup for this should be in the Accessibility tab of the admin settings for an app.
- The new properties should be added to the Additional Information section of the App Details page.

The updated Accessibility attribute is [here](http://imsglobal.github.io/casa-protocol/#Attributes/Accessibility:wcag_compliance).
